### DISCUSSION TITLE
my iphone is dead

### DISCUSSION THREAD

**Discussion by itshak770** [Authenticator-Extension/Authenticator#617] [General]
My Iphone is dead so I can't acces to Authenticator and then I can't access to the applications without an Authenticator code.

What can I do ?

**Comment by mymindstorm** [comment]
Is this question only related to the iPhone app?

**Comment by itshak770** [comment]
yes

 

De : mymindstorm ***@***.*** 
Envoyé : lundi 22 mars 2021 21:56
À : Authenticator-Extension/Authenticator
Cc : itshak770; Author
Objet : Re: [Authenticator-Extension/Authenticator] my iphone is dead (#616)

 

Is this question only related to the iPhone app?

**Comment by Sneezry** [comment]
Hi @itshak770 , this project for is Authenticator Extension, which runs in browser. If your issue is about Google official Authenticator iOS app, the right place you are looking for is https://github.com/google/google-authenticator
